redisinsight/ui/src/pages/rdi/home/RdiPage.tsx

```diff
@@ -1,87 +1,140 @@
-import { EuiPage, EuiPageBody, EuiPanel, EuiResizeObserver } from '@elastic/eui'
+import { EuiPage, EuiPageBody, EuiPanel, EuiResizableContainer, EuiResizeObserver } from '@elastic/eui'
 import React, { useEffect, useState } from 'react'
 import { useDispatch, useSelector } from 'react-redux'
 
+import { RdiInstance } from 'uiSrc/slices/interfaces'
 import {
   createInstanceAction,
+  editInstanceAction,
   fetchInstancesAction,
-  instancesSelector,
-  setEditedInstance
+  instancesSelector
 } from 'uiSrc/slices/rdi/instances'
+import { TelemetryEvent, sendEventTelemetry } from 'uiSrc/telemetry'
 import EmptyMessage from './components/EmptyMessage'
+import ConnectionForm from '../connection-form/ConnectionForm'
 import RdiHeader from '../header/RdiHeader'
 import RdiInstancesListWrapper from '../instance-list/RdiInstancesListWrapper'
 
 import styles from './styles.module.scss'
 
-export interface Props {}
-
 const RdiPage = () => {
   const [width, setWidth] = useState(0)
+  const [isConnectionFormOpen, setIsConnectionFormOpen] = useState(false)
+  const [editInstance, setEditInstance] = useState<RdiInstance | null>(null)
 
-  const {
-    data,
-    loading,
-    loadingChanging
-  } = useSelector(instancesSelector)
+  const { data, loading, loadingChanging } = useSelector(instancesSelector)
 
   const dispatch = useDispatch()
 
   useEffect(() => {
     dispatch(fetchInstancesAction())
-
-    return () => {
-      dispatch(setEditedInstance(null))
-    }
   }, [])
 
   const onResize = ({ width: innerWidth }: { width: number }) => {
     setWidth(innerWidth)
   }
 
-  const handleAddInstance = () => {
-    dispatch(
-      createInstanceAction({
-        name: 'My first integration',
-        url: 'redis-12345.c253.us-central1-1.gce.cloud.redislabs.com:12345',
-        version: '1.2',
-        username: 'username',
-        password: 'password'
-      })
-    )
-    dispatch(fetchInstancesAction())
-    dispatch(setEditedInstance(null))
+  const handleAddInstance = (instance: Partial<RdiInstance>) => {
+    const name = instance.name === '' ? instance.url : instance.name
```

Comment:
May be better to call it alias? (to be consistence with databases and with field name)

Reply:
`name` is what the backend calls it as well so if we rename it here I assume we would also want to rename it in the backend. Thoughts?

Reply:
Lets keep it as 'name'